Component won't install on React 17.

It looks like when you set the "minimum" version to 16.3.0 you did it incorrectly.   `^16.3.0` means any version of 16... which means react 17 doesn't count and so you get an error trying to install:

```% npm install react-map-interaction                                                                                         21-03-22 - 15:48:33
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: starter@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0" from react-map-interaction@2.0.0
npm ERR! node_modules/react-map-interaction
npm ERR!   react-map-interaction@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/echo/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/echo/.npm/_logs/2021-03-22T20_48_36_892Z-debug.log```

--- a/changelog.md
+++ b/changelog.md
@@ -1,5 +1,7 @@
 # Unreleased
 
+Add support for React 17.
+
 # 2.0.0
 
 BREAKING: To make compatible with React 17, we got rid of componentWillReceiveProps usage. In doing so,

--- a/example/package.json
+++ b/example/package.json
@@ -6,8 +6,8 @@
     "react-scripts": "^2.1.3"
   },
   "dependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
   },
   "scripts": {
     "start": "react-scripts start",

--- a/package.json
+++ b/package.json
@@ -25,8 +25,8 @@
   "private": false,
   "license": "MIT",
   "peerDependencies": {
-    "react": "^16.3.0",
-    "prop-types": "^15.0.0"
+    "react": ">=16.3.0",
+    "prop-types": ">=15.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.3.3",

